src/assets/styles/_popper.scss

```diff
@@ -111,7 +111,7 @@ $pointer_offset: -$pointer_size/2;
     position: absolute;
     width: $pointer_size;
     height: $pointer_size;
-    z-index: -1;
+    z-index: 1;
```

Comment:
shouldn't the tail be under?

Reply:
It was, but the border went over the pointer and looks disconnected. 
![image](https://user-images.githubusercontent.com/6144462/111369261-186b6b80-8654-11eb-94b8-12efe7167dad.png)

Reply:
The tooltip didn't have a border before, which is why it wasn't noticeable, but from the design, it does need a border.